enterprise/pageembed.md

```diff
@@ -17,13 +17,13 @@ Page Embed provides more flexibility than an inline script and can contain an en
 
 #### Creating an account
 
-To try out Page Embed and Tiny Cloud-delivered editor, the first step is to create a free [Tiny account](https://www.tiny.cloud/download/).
+To try out Page Embed and {{site.cloudname}}-delivered editor, the first step is to create a free [{{site.accountpage}}](https://www.tiny.cloud/download/).
 
-> The free user account provisioned with a free 30-day trial of all Tiny [premium plugins](https://apps.tiny.cloud/product-category/tiny-cloud-extensions/), with no credit card information or commitment required.
+> The free {{site.accountpage}} is provisioned with a free 30-day trial of all {{site.cloudname}} [premium plugins]({{site.cloudextensions}}), with no credit card information or commitment required.
```

Comment:
One more `Tiny Account` that doesn't quite seem to read right?

Reply:
Changed to `{{site.accountpage}}s receive a free 30-day` in all instances